### PR TITLE
Make public autofocus field of the TextField of message_input

### PR DESCRIPTION
**Pull request by TramPamPam**
# Submit a pull request

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x] The code changes follow best practices
- [x] Code changes are tested (add some information if not applicable)

### CODE REVIEW

**Comment by TramPamPam** [comment]
@imtoori @xsahil03x @tbarbugli 

Could someone review this pull request, please?

File: lib/src/message_input.dart [hunk 1]
```diff
@@ -258,7 +262,7 @@ class MessageInputState extends State<MessageInput> {
             });
           },
           style: widget.inputTextStyle ?? Theme.of(context).textTheme.bodyText2,
-          autofocus: false,
+          autofocus: widget.autofocus ?? false,
```
Comment (xsahil03x) [inline lib/src/message_input.dart:265]: Can you please set the default value in the constructor itself. We do not really need to handle the null case as the textField itself throws an assertion error.

**Review by xsahil03x — CHANGES_REQUESTED** [review]
@TramPamPam Just left one comment otherwise LGTM.

File: lib/src/message_input.dart (hunk 1 above)
Comment (imtoori) [reply lib/src/message_input.dart:265]: I'll merge it and add this change

**Review by imtoori — APPROVED** [review]

File: lib/src/message_input.dart (hunk 1 above)
Comment (TramPamPam) [reply lib/src/message_input.dart:265]: Sure thing. Changed in the last commit, please check.

**Pull request merged**